src/askui/tools/askui/askui_controller.py

```diff
@@ -723,6 +730,14 @@ def set_display(self, display: int = 1) -> None:
         )
         self._display = display
 
+    @telemetry.record_call()
+    @override
+    def get_active_display(self) -> int:
+        """
+        Get the active display.
+        """
+        return self._display

```

Comment:
Well done :) I think we should either include this info in the display information returned by the listing tool / the `GetDisplayInformationResponse` or add an extra tool for this. 

I think it would be great if this was included in the controller api as I think they maintain the source of truth about which display is currently active (see https://askyourui.slack.com/archives/C091TSQ6KP0/p1753431836333729).